README.md

```diff
@@ -134,6 +134,14 @@ cmake --build build -j 8
 ```
 This way, all the functionalities are executed on A100 GPUs. For installation, details are provided in the [documentation](https://kokkosfft.readthedocs.io/en/latest/intro/building.html#install-kokkosfft-as-a-library).
 
+## Support
+
+Find us on Slack: https://kokkosteam.slack.com #kokkos-fft or open a GitHub issue.
```

Comment:
We can put the link the direct link to the channel if you want https://kokkosteam.slack.com/archives/C07NM5SD3FD